example/lazy/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
This isn't linked up to the root `index.html`, so you have to manually navigate to it! I think we should add a link since the other stuff is all linked from there

Reply:
I was actually not showing it quite yet just because `lazy` isn't really like "public" yet... But i guess i could be convinced since it demos suspense... I honestly even considered not including this but i used to for manual testing... and thought it would be a shame to lose it lol

Reply:
It's as much a `Suspense` demo as it is a lazy one, and `Suspense` is public 🙂 I think it also just makes our life harder  as well, so I'd still vote for it, but it's probably not worth blocking the PR over!